.github/workflows/_build.yml

```diff
@@ -96,8 +95,7 @@ jobs:
       - name: Build
         shell: bash
         run: |
-          docker exec -u jenkins "${pid}" bash -c ". ~/.bashrc && .circleci/build.sh"
```

Comment:
"~/.bashrc" can be removed because it has been done in the dockerfile?

Reply:
correct